### ISSUE TITLE
[UX] Hover states and colours

### ISSUE THREAD

**Issue by jonbell-lot23** [RecordReplay/devtools#1746]
The first big chunk of this is being worked through by Jaril and Jon in Figma, as part of Project Left Wing.

We'll open a separate ticket for additional colour work once this is done.

[Figma link](https://www.figma.com/file/SSr1ljyzF0lCXTL7BX1Whh/Replay-Design-Doc?node-id=2313%3A2510) -- Look for page States and Colours

**Comment by jaril** [comment]
This was done in #1762.
